crossbuilder

```diff
@@ -1040,3 +1043,6 @@ else
         ;;
     esac
 fi;
+
+#reset locale
+LC_ALL=
```

Comment:
Remove this part, it's not needed

Reply:
https://github.com/ubports/crossbuilder/commit/cf5ff543dbe3b37dd12361f983b923d4be72e30e